src/server/macos.rs

```diff
@@ -424,29 +435,49 @@ impl<'a> Instance for LocalInstance<'a> {
         let service = launchctl_status(&self.name, system,
             // TODO
             &StatusCache::new());
-        let service_exists = launchd_plist_path(&self.name, system)
-            .map(|p| p.exists())
+        let service_exists = self.get_start_conf()
+            .map(|start_conf| launchd_plist_path(
+                &self.name, system, start_conf == StartConf::Auto)
+                .map(|p| p.exists())
+                .unwrap_or(false)
+            )
             .unwrap_or(false);
         unix::status(&self.name, &self.path, service_exists, service)
     }
     fn start(&self, options: &Start) -> anyhow::Result<()> {
         if options.foreground {
             process::run(&mut self.get_command()?)?;
         } else {
-            let lname = self.launchd_name();
-            process::run(
-                StdCommand::new("launchctl").arg("enable").arg(&lname)
-            )?;
-            process::run(
-                StdCommand::new("launchctl").arg("kickstart").arg(&lname)
-            )?;
+            match launchctl_status(&options.name, false, &StatusCache::new()) {
+                Service::Running {..} => {
+                    log::error!(target: "edgedb::server::start",
```

Comment:
It's not an error running `start` if service is already started. We can write this on `info` level, I think even `warn` is too scary.

Reply:
OK!